google-api/googleapis.js

```diff
@@ -14,6 +14,57 @@ const oauth2Client = new google.auth.OAuth2(
 
 oauth2Client.setCredentials({ refresh_token: GOOGLE_REFRESH_TOKEN })
 
+const dedupeTime = (items) => {
+  const deduped = items.reduce((
+    acc,
+    {
+      summary,
+      start: { dateTime: startDate },
+      end: { dateTime: endDate },
+      htmlLink
+    },
+    i) => {
+    const eventKey = `event${i}`
+    const  existingEvents = Object.entries(acc || {})
+    let replaced = false
+    if (existingEvents.length) {
+      existingEvents.forEach(([key, { start, end }]) => {
+        const s = startDate >= start && startDate <= end
+        const e = endDate >= start && endDate <= end
```

Comment:
not sure if the comparison between date strings `"2020-12-21T10:30:00-05:00"` (api response) are ok or if I should `new Date("2020-12-21T10:30:00-05:00").getTime()` here to play safe

Reply:
if without the time zone then i'm 100% that'd be safe. with time zone not so much. perhaps use getTime for safety